server/repository/src/migrations/v1_06_00/encounter_add_clinician_link_id.rs

```diff
@@ -0,0 +1,40 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    #[cfg(feature = "postgres")]
+    sql!(
+        connection,
+        r#"
+            ALTER TABLE encounter
+            ADD COLUMN clinician_link_id TEXT;
+
+            UPDATE encounter SET clinician_link_id = clinician_id;
+
+            ALTER TABLE encounter ADD CONSTRAINT encounter_clinician_link_id_fkey FOREIGN KEY (clinician_link_id) REFERENCES clinician_link(id);
```

Comment:
do we need to add index here ?

Reply:
Moving index sql from the not postgres block to the general block at the bottom, will do for the other migrations too!

Reply:
On second thought I'll look tomorrow - should check the FK constraint syntax for pgsql doesn't automatically generate the index. Though if it doesn't then surely we don't need this to run for postgres either, or at least all the ones for FKs. 

https://github.com/msupply-foundation/open-msupply/blob/a025f2405bf5e9d7b6f4790b080fcdf571ab83ef/server/repository/src/migrations/v1_06_00/indexes.rs#L3-L8

Reply:
It doesn't. I'm noticing a bunch of mistakes like this across the migrations for merge, and possibly other mistakes too. I'll fix this one here but open an issue for the rest.